### PR TITLE
Add start of status screen and add new picked up button

### LINKED ISSUE #519: Make McDonald's status screen

### PR DESCRIPTION
**Pull request by KiOui**
Closes #519

### CODE REVIEW

**Review by JobDoesburg — COMMENTED** [review]
Looks good. Maybe we should put the status screen in a separate app though. I think that would help keep the orders app small (its already a bit too large)

**Comment by KiOui** [comment]
Shall I merge this?

**Comment by JobDoesburg** [comment]
@KiOui let's separate the status screen

**Comment by KiOui** [comment]
Updated!

File: website/status_screen/urls.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from status_screen import views
+
+urlpatterns = [
+    path("<shift:shift>/", views.StatusScreen.as_view(), name="status"),
```
Comment (JobDoesburg) [inline website/status_screen/urls.py:6]: Maybe we should add a path with a venue, that automatically redirects to the currently open shift in that venue.

File: website/status_screen/templates/status_screen/status_screen.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                },
+                refresh() {
+                    if (this.refreshing) {
+                        return;
+                    }
+
+                    clearTimeout(this.refreshTimer);
+                    this.refreshing = true;
+                    return fetch(
+                        `/api/v1/shifts/{{ shift.id }}/orders/?type=0&picked_up=false&ordering=ready_at`,
```
Comment (JobDoesburg) [inline website/status_screen/templates/status_screen/status_screen.html:158]: Is there maybe a way to use a {% url %} template tag here?

Like in https://github.com/JobDoesburg/yivi-portal/blob/51ced900cbda534e763f61403eb27eb5251740e6/schememanager/templates/yivi_request.html#L36

Comment (JobDoesburg) [reply website/status_screen/templates/status_screen/status_screen.html:158]: And similarly on all other places, actually

**Review by JobDoesburg — CHANGES_REQUESTED** [review]

File: website/status_screen/templates/status_screen/status_screen.html (hunk 2 above)
Comment (JobDoesburg) [reply website/status_screen/templates/status_screen/status_screen.html:158]: @KiOui status update?

**Review by JobDoesburg — APPROVED** [review]

**Pull request merged**